Dockerfile

```
@@ -5,12 +5,13 @@
 FROM openjdk:11-jdk-slim AS java-builder
 
 # tesseract-ocr < 4.0 is only available from the previous Debian Stretch distribution
-RUN echo "deb http://ftp.de.debian.org/debian stretch main" >> /etc/apt/sources.list
+# for installing it plese uncomment the following lines with '###''
+### RUN echo "deb http://ftp.de.debian.org/debian stretch main" >> /etc/apt/sources.list
 
 RUN apt-get update && \
 #	apt-get dist-upgrade -y && \
-#	apt-get install -y tesseract-ocr && \
-	apt-get install -y tesseract-ocr-osd=3.04.00-1 tesseract-ocr-eng=3.04.00-1 tesseract-ocr=3.04.01-5 && \
+	apt-get install -y tesseract-ocr && \
```

It may be better practice to always specify versions of applications installed so the builds of the container behave consistently across different `docker build`
```suggestion
	apt-get install -y tesseract-ocr && \
```